pkg/kubelet/pluginmanager/pluginwatcher/plugin_watcher.go

```diff
@@ -202,6 +208,12 @@ func (w *Watcher) handleCreateEvent(event fsnotify.Event) error {
 }
 
 func (w *Watcher) handlePluginRegistration(socketPath string) error {
+	if runtime.GOOS == "windows" {
+		socketPath = strings.Replace(socketPath, "/", "\\", -1)
+		if strings.HasPrefix(socketPath, "\\") {
+			socketPath = "c:" + socketPath
+		}
+	}
```

Comment:
Thanks for highlighting the common functionality - one note, it makes me a little nervous to define a function in `pkg/kubelet/util/util.go` that is then imported by `pkg/volume` and `pkg/util/mount`.

Reply:
+1 for factoring the function out. Maybe we should have a `pkg/util/windows` package.

For this PR, I think we can put the helper function in `pkg/kubelet/util` so that we don't get more code duplication within kubelet.

Reply:
Done. Moved to pkg/kubelet/util for now.